lib/registration.pm

```diff
@@ -119,12 +135,20 @@ sub fill_in_registration_data {
                     save_screenshot;
                 }
             }
-            send_key 'alt-n';    # next
-            while (check_screen('import-untrusted-gpg-key', 20)) {
-                record_soft_failure 'untrusted gpg key';
-                send_key 'alt-t', 2;
+            send_key 'alt-n';
+            # start addons/modules registration, it needs longer time if select multiple or all addons/modules
+            while (check_screen(['import-untrusted-gpg-key', 'yast_scc-pkgtoinstall', 'inst-addon'], 120)) {
```

Comment:
@mitiao maybe you missed my previous comment:

Something like this should work and not waste any time:
```perl
assert_screen([qw/scc-finishes-correctly scc-comes-up-with-surprises/]);
if (match_has_tag('scc-comes-up-with-surprises')) {
    # do something special
    assert_screen 'scc-finishes-correctly';
}
```